src/util/event-to-promise.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
```

Comment:
Are you referring to the filter tests? The filter option is documented:

```
// - filter (optional):
//     Supply a predicate function to decide for each event whether to
//     resolve/reject or to ignore it. The function gets passed the event
//     parameters, as if it was an event handler.
```

So tests would be:

* Can I selectively ignore certain events for resolving
* Can I selectively ignore certain events for rejecting